dts/bindings/spi/renesas,smartbond-spi.yaml

```diff
@@ -18,3 +18,17 @@ properties:
     type: int
     description: Peripheral clock register configuration (COM domain)
     required: true
+
+  tx-dma-channel:
+    type: int
+    description: |
+      DMA channel to be reserved for SPI TX trasmissions.
+      It is imperative that a TX channel be assigned
+      an odd number that is [1, 3, 5, 7].
+
+  rx-dma-channel:
```

Comment:
this should be coming from a dmas property, why does smartbond DMA not use #dma-cells?

Reply:
You are right. DMA channels and controller are now retrieved via `DT_INST_DMAS_CELL_BY_NAME` and `DT_INST_DMAS_CTLR_BY_NAME` macros respectively.